Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocList.css

```diff
@@ -377,6 +378,7 @@ div.bocList table.bocListTable tr.bocListValidationRow.selected td
 
 div.bocList div.bocListTableBlock tr.bocListValidationRow td ul
 {
+  padding-left: var(--bocList-validationrow-ul-padding);
```

Comment:
would like to do a quick live demo once the changes are done to check how things look with the various column configurations.